### DISCUSSION TITLE
用户名称附近有没有牌子的显示方法？

### DISCUSSION THREAD

**Discussion by stdue** [xfgryujk/blivechat#156] [提问] [answered]
查看过wiki和样式生成器没发现关于牌子的显示和相应的样式修改，这一部分应该怎么处理？

**Comment by xfgryujk** [comment, marked as answer]
用魔改版

**Comment by stdue** [comment]
@xfgryujk 魔改版能帮忙指一下路吗，谢谢了🙏

**Comment by xfgryujk** [comment]
> @xfgryujk 魔改版能帮忙指一下路吗，谢谢了🙏

https://github.com/DoodleBears/blivechat
